### ISSUE TITLE
widget:ViewResize is not called if the game is started directly after a view change

### ISSUE THREAD

**Issue by GoogleFrog** [Spring-Chobby/Chobby#272]
The lobby panels do not update when a game starts (with Display set to autodetect). This implies that widget:ViewResize is not being called for the start-of-game resize. The output of text to infolog seems too unreliable to test this theory directly.

**Comment by GoogleFrog** [comment]
Actually, not fixed.

**Comment by GoogleFrog** [comment]
I am fairly convinced that widget:ViewResize does not call on game start. The tooltip widget indicates that this is the case.

**Comment by ashdnazg** [comment]
you are correct, the engine gets this event during some limbo where it can't send it to luamenu.
Since you change the window stuff yourself, you can probably call it manually
